qr_generator/lib/model/seed_model.dart

```diff
@@ -0,0 +1,30 @@
+class SeedModel {
+  SeedModel({
+    this.id,
+    this.expiresAt,
+  });
+
+  final String id;
+  final DateTime expiresAt;
+
+  SeedModel copyWith({
+    String id,
+    DateTime expiresAt,
+  }) =>
+      SeedModel(
+        id: id ?? this.id,
+        expiresAt: expiresAt ?? this.expiresAt,
+      );
+
+  factory SeedModel.fromJson(Map<String, dynamic> json) => SeedModel(
```

Comment:
**question:** have you used json serialization tools with Dart & Flutter?

Reply:
Usually, I do manual serialization. But after reading about it today, I'll change it using code generation libraries.